components/collector/__init__.py

```diff
@@ -484,21 +484,3 @@ def available_resources(site: Site,
                 Resource(id='H'),
                 Resource(id='J')
             }))
-
-    def get_actual_conditions_variant(self,
-                                      site: Site,
-                                      times: Time) -> Optional[Variant]:
-        """
-        Return the weather variant.
-        This should be site-based and time-based.
-        """
-        night_length = len(times)
-
-        return Variant(
-            iq=np.full(night_length, ImageQuality.IQ70),
-            cc=np.full(night_length, CloudCover.CC50),
-            wv=np.full(night_length, WaterVapor.WVANY),
-            wind_dir=Angle(np.full(night_length, 330.0), unit='deg'),
-            wind_sep=Angle(np.full(night_length, 40.0), unit='deg'),
-            wind_spd=Quantity(np.full(night_length, 5.0 * u.m / u.s))
-        )
```

Comment:
😄 This was the next task I was going to give you. I don't know how VSC works, so I don't know if it tells you what imports you don't need any more, but there should be imports that are not being used any longer in this file. Could you remove them? (That was the second part of the next task.)